common/thorhelper/thorxmlwrite.cpp

```diff
@@ -70,6 +70,19 @@ void CommonXmlWriter::closeTag()
     flush(false);
 }
 
+size32_t CommonXmlWriter::checkCloseLength(unsigned len, const char* data)
```

Comment:
@mayx I'd like to try to think of a better abstraction for what we're really trying to do here.  May be tricky.